spec/calendar_spec.rb

```diff
@@ -33,6 +33,14 @@
     end
   end
 
+  describe '.each' do
+    it 'yields calendar day instances' do
```

Comment:
Please add another test checking that `each` actually yields. This one only checks what it yields *if* it yields.